ci-operator/step-registry/ipi/OWNERS

```diff
@@ -8,3 +8,4 @@ approvers:
 - ewolinetz
 - csrwng
 - staebler
+- technical-release-team
```

Comment:
Does it make sense to drop TRT in here, or one level up to cover the whole step registry?

Reply:
I'd prefer to keep it here. We can rearrange owners when TRT is ready to handle the whole step registry